api_calls/reference/inquiries/inquiries_read-inquiries_write-inquiries_write_owned-scope-response.json

```diff
@@ -18,8 +22,11 @@
       "adults": 4,
       "children": 2,
       "message": "Important message",
+      "locked": null,
       "created_at": "CREATED_AT",
       "updated_at": "UPDATED_AT"
     }
-  ]
+  ],
+  "meta": {
```

Comment:
hmm I am not sure there is `meta` returned for this endpoint.

btw why do we need other responses?